### ISSUE TITLE
Some cong tests in error

### ISSUE THREAD

**Issue by naipmoro** [gap-packages/Semigroups#227]
Hello,
I am on ubuntu 16.04, using the latest 4.8.5 gap, which I compiled. All the recent 4.8.x
versions of _semigroups-2.8.0_ give the same test errors in simple-cong and reesmat-cong:

 ```Running list 195 . . .
 Diff in 
[ 
  "/home/naip/apps/gap/gap4r8p5/pkg/semigroups-2.8.0/doc/simple-cong.xm\
l", 52, 63 ]
 **Input is:**
congs := CongruencesOfSemigroup(s);
 **Expected output:**
[ <universal semigroup congruence over 
    <Rees 0-matrix semigroup 2x2 over Sym( [ 1 .. 3 ] )>>, 
  <semigroup congruence over <Rees 0-matrix semigroup 2x2 over 
      Sym( [ 1 .. 3 ] )> with linked triple (1,2,2)>, 
  <semigroup congruence over <Rees 0-matrix semigroup 2x2 over 
      Sym( [ 1 .. 3 ] )> with linked triple (3,2,2)>, 
  <semigroup congruence over <Rees 0-matrix semigroup 2x2 over 
      Sym( [ 1 .. 3 ] )> with linked triple (S3,2,2)> ]
 **But found:**
[ <universal semigroup congruence over 
    <Rees 0-matrix semigroup 2x2 over Sym( [ 1 .. 3 ] )>>, 
  <semigroup congruence over <Rees 0-matrix semigroup 2x2 over 
      Sym( [ 1 .. 3 ] )> with linked triple (S3,2,2)>, 
  <semigroup congruence over <Rees 0-matrix semigroup 2x2 over S3>
      with linked triple (3,2,2)>, 
  <semigroup congruence over <Rees 0-matrix semigroup 2x2 over S3>
      with linked triple (1,2,2)> ]

 Running list 204 . . .
 Diff in 
[ 
  "/home/naip/apps/gap/gap4r8p5/pkg/semigroups-2.8.0/doc/reesmat-cong.x\
ml", 282, 288 ]
 **Input is:**
MeetSemigroupCongruences(congs[2], congs[3]);
 **Expected output:**
<semigroup congruence over <Rees 0-matrix semigroup 2x2 over 
  Sym( [ 1 .. 3 ] )> with linked triple (1,2,2)>
 **But found:**
<semigroup congruence over <Rees 0-matrix semigroup 2x2 over 
  Sym( [ 1 .. 3 ] )> with linked triple (3,2,2)>

 Running list 205 . . .
 Diff in 
[ 
  "/home/naip/apps/gap/gap4r8p5/pkg/semigroups-2.8.0/doc/reesmat-cong.x\
ml", 303, 309 ]
**Input is:**
JoinSemigroupCongruences(congs[2], congs[3]);
**Expected output:**
<semigroup congruence over <Rees 0-matrix semigroup 2x2 over 
  Sym( [ 1 .. 3 ] )> with linked triple (3,2,2)>
**But found:**
<semigroup congruence over <Rees 0-matrix semigroup 2x2 over 
  Sym( [ 1 .. 3 ] )> with linked triple (S3,2,2)>```

**Comment by mtorpey** [comment]
Thanks for posting this!  It looks like it's just a change in the order of a list, which might be caused by GAP's function for finding normal subgroups of a group.  I'll try to reproduce this today, and amend code/tests as appropriate.

**Comment by james-d-mitchell** [comment]
Thanks for posting this indeed! We'll try to get it fixed as soon as possible.

**Comment by mtorpey** [comment]
I've fixed the issue in [pull request #228](https://github.com/gap-packages/Semigroups/pull/228).

The issue was caused by a change in the order of the list given by CongruencesOfSemigroup for a Rees matrix semigroup. As such, it was a harmless diff, and I've modified the tests so that they don't rely on ordering.

The fix is to be merged into `stable-2.8`, so should be included in the next release.

**Comment by mtorpey** [comment]
This is fixed in the upcoming v3.0, so we can close this issue as soon as that release is made (hopefully this week).

**Comment by mtorpey** [comment]
This is now fixed in v3.0.0, so this issue can be closed.
